webroot/src/pages/Login/Login.ts

```diff
@@ -24,7 +24,7 @@ export default class Login extends Vue {
 
     get hostedLoginUriStaff(): string {
         const { domain, cognitoAuthDomainStaff, cognitoClientIdStaff } = this.$envConfig;
-        const loginScopes = 'email openid phone profile';
+        const loginScopes = 'email openid phone profile aws.cognito.signin.user.admin';
```

Comment:
What's this new scope do/enable?

Reply:
It gives cognito REST API access to the auth token. For instance, to allow the user to call the change-password API.